another.c

```diff
@@ -1 +1,2 @@
-another file from tutorial
\ No newline at end of file
+another file from tutorial
+editted by contributer
```

Comment:
what a poor code you idiot! 
fall to hell!! and never get back!!

Reply:
are you serious?